lms/views/helpers/_via.py

```diff
@@ -20,6 +20,13 @@ def via_url(request, document_url, content_type=None):
     """
     return ViaClient(
         service_url=request.registry.settings["via_url"],
-        host_url=request.host_url,
         secret=request.registry.settings["via_secret"],
-    ).url_for(document_url, content_type, blocked_for="lms")
+    ).url_for(
+        document_url,
+        content_type,
+        blocked_for="lms",
+        options={
+            "via.client.requestConfigFromFrame.origin": request.host_url,
+            "via.client.requestConfigFromFrame.ancestorLevel": "2",
+        },
+    )
```

Comment:
There are actually are [tests for this](https://github.com/hypothesis/lms/blob/2062c02df31287fee0ee81e778cc6a3280c8a4d7/tests/unit/lms/views/helpers/_via_test.py#L16-L66) but the tests didn't need to be changed, which is good. If you remove or break one of the `via.client.requestConfigFromFrame.*` options those tests fail